static-site/content/blog/2024-06-18-h5n1-cattle-outbreak-analysis-and-resources.md

```diff
@@ -0,0 +1,132 @@
+
+At the moment, USDA are first sharing raw sequence reads to SRA before sharing assembled consensus genomes to GenBank. Unfortunately these SRA entries lack metadata on time and place beyond “2024” and “USA”, which [significantly impedes their utility for epizootic understanding](https://www.statnews.com/2024/05/02/bird-flu-in-cows-h5n1-virus-changes-missing-data/).
+Additionally, the process of assembly is computationally intensive and it’s unwieldy for everyone analyzing these data to need to work from raw reads.
+This is a significant issue as, as of this writing, there are [293 consensus genomes from USDA available via GenBank](https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/virus?SeqType_s=Nucleotide&EnvSample_s=include&VirusLineage_ss=Alphainfluenzavirus,%20taxid:197911&Authors_idx%20q.op%3DAND=*&SubmitterAffil_idx%20q.op%3DOR=USDA-ARS&Serotype_s=H5N1%20H5N*&CollectionDate_dr=2023-12-31T00:00:00.00Z%20TO%202024-06-17T23:59:59.00Z&SLen_i=2280%20TO%203000000), while there are [653 samples with sequencing reads in the SRA](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1102327).
+Almost half the dataset is missing critical metadata and the most recently available consensus genomes were from samples collected in early April about two months ago.
+
+To ameliorate this issue, Kartik Gangavarapu, Praneeth Gangavarapu and Kristian Andersen at Scripps have separately taken raw read data from the SRA, assembled it into consensus genomes and shared the resulting FASTA files via [GitHub](https://github.com/andersen-lab/avian-influenza).
+During our ingest process, we merge these assembled consensus genomes with consensus genomes available via GenBank. We deduplicate between data sources based on SRA accession.
+
+GenBank was updated with 280 genomes on May 3 by the USDA corresponding to data described in the [Nyugen et al preprint](https://www.biorxiv.org/content/10.1101/2024.05.01.591751v1), but following this there have been only 27 genomes shared to GenBank by the USDA since May 3 (**Fig. 7**).
+However, sharing to the SRA has continued with 393 submissions since May 3.
+
+![fig7](img/h5n1_cattle_outbreak_fig7_genbank_vs_sra_release_date.png)
+**Figure 7. Release dates for H5N1 genomes from the US in GenBank and in the SRA.**
+
+This [ingest pipeline](https://github.com/nextstrain/avian-flu/tree/master/ingest) is [run daily](https://github.com/nextstrain/avian-flu/actions/workflows/ingest-ncbi.yaml) and resulting sequences files are available as:
+- [data.nextstrain.org/files/workflows/avian-flu/h5n1/ha/sequences.fasta.zst](https://data.nextstrain.org/files/workflows/avian-flu/h5n1/ha/sequences.fasta.zst)
+- [data.nextstrain.org/files/workflows/avian-flu/h5n1/na/sequences.fasta.zst](https://data.nextstrain.org/files/workflows/avian-flu/h5n1/na/sequences.fasta.zst)
+- etc...
```

Comment:
In most contexts where "etc" appears in this post, the reader could somehow get to other genes (e.g., by clicking on the "ha" tree link and finding the other gene segments in the Auspice data dropdown), but here I think readers will be especially confused about how to find the other gene sequences. With an AWS resource, a saavy reader could figure out the `aws s3 ls` command to find the other gene segments, but we don't provide that kind of listing for data.nextstrain.org yet. The mapping of standard gene names to the lowercase versions we use isn't always obvious (matrix protein could be "mp", "ma", "m1", "m2").

I propose adding a line for each gene's sequence here:

```suggestion
- [data.nextstrain.org/files/workflows/avian-flu/h5n1/mp/sequences.fasta.zst](https://data.nextstrain.org/files/workflows/avian-flu/h5n1/mp/sequences.fasta.zst)
- [data.nextstrain.org/files/workflows/avian-flu/h5n1/np/sequences.fasta.zst](https://data.nextstrain.org/files/workflows/avian-flu/h5n1/np/sequences.fasta.zst)
- [data.nextstrain.org/files/workflows/avian-flu/h5n1/ns/sequences.fasta.zst](https://data.nextstrain.org/files/workflows/avian-flu/h5n1/ns/sequences.fasta.zst)
- [data.nextstrain.org/files/workflows/avian-flu/h5n1/pa/sequences.fasta.zst](https://data.nextstrain.org/files/workflows/avian-flu/h5n1/pa/sequences.fasta.zst)
- [data.nextstrain.org/files/workflows/avian-flu/h5n1/pb1/sequences.fasta.zst](https://data.nextstrain.org/files/workflows/avian-flu/h5n1/pb1/sequences.fasta.zst)
- [data.nextstrain.org/files/workflows/avian-flu/h5n1/pb2/sequences.fasta.zst](https://data.nextstrain.org/files/workflows/avian-flu/h5n1/pb2/sequences.fasta.zst)
```

Reply:
Good call! Especially nice here to list things out in that it should make the subsequent use of "etc..." in the following sections more obvious. Resolved in de14773b68f03ec6c5379397e817e7d37a009cc9.